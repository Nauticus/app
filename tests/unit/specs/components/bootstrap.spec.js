import {
    shallowMount,
    createLocalVue,
} from '@vue/test-utils';
import VueRouter from 'vue-router';

import SBootstrap from '@/components/bootstrap.vue';


describe('Component `bootstrap`', () => {
    it('should have correct name', () => {
        expect(SBootstrap.name).toBe('SBootstrap');
    });

    it('should have correct markup', () => {
        const localVue = createLocalVue();
        localVue.use(VueRouter);

        const wrapper = shallowMount(SBootstrap, {
          localVue
        });

        expect(wrapper.element).toMatchSnapshot();
    });
});
