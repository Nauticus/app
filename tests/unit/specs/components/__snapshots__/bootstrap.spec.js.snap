// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component \`bootstrap\` should have correct markup 1`] = `
<div
  class="s-bootstrap"
>
  <router-view-stub
    name="default"
  />
</div>
`;
